docs/pages/developer-guide.md

```diff
@@ -1512,7 +1512,6 @@ DefaultServiceSpec.newBuilder()
     .name(FRAMEWORK_NAME)
     .role(ROLE)
     .principal(PRINCIPAL)
-    .apiPort(8080)
```

Comment:
That setting had been removed from DefaultServiceSpec previously, I think this is just cleaning up a leftover reference